templates/plugin/gutterball/foreman_gutterball_url.yaml.erb

```diff
@@ -0,0 +1,6 @@
+:foreman_gutterball_url:
+  :protocol: <%= @protocol %>
```

Comment:
@komidore64 do you think there is any value for foreman-gutterball to have discrete portions of the url?